proxmox/resource_vm_qemu.go

```diff
@@ -897,10 +903,19 @@ func resourceVmQemuCreate(d *schema.ResourceData, meta interface{}) error {
 	// give sometime to proxmox to catchup
 	//time.Sleep(time.Duration(d.Get("additional_wait").(int)) * time.Second)
 
-	log.Printf("[DEBUG][QemuVmCreate] starting VM: %d", vmr.VmId())
-	_, err := client.StartVm(vmr)
-	if err != nil {
-		return err
+	if d.Get("oncreate").(bool) {
+		log.Print("[DEBUG][QemuVmCreate] starting VM")
+		_, err := client.StartVm(vmr)
```

Comment:
Careful with the scope of err, you'll see some build/test errors now

Reply:
i was able to build it locally. do i need to use = instead of := there for err, or does something else need moved around?